lia-demo/src/dev/java/com/lithium/community/android/example/InitializationActivity.java

```diff
@@ -164,7 +165,22 @@ private void persistAndUpdate() {
         // TODO: save the credentials in shared preferences
         // TODO: logout only if credentials have changes since last session.
         if (LiSDKManager.isInitialized() && LiSDKManager.getInstance().isUserLoggedIn()) {
-            LiSDKManager.getInstance().logout(this);
+            LiSDKManager.getInstance().logout(this, new Callback<Void, Throwable>() {
+                @Override
+                public void success(Void aVoid) {
+
+                }
+
+                @Override
+                public void failure(Throwable throwable) {
+
+                }
+
+                @Override
+                public void abort(Throwable throwable) {
+
+                }
+            });
```

Comment:
Maybe add a toast in the callback.

Reply:
this call itself is not needed, previously I think this was done as a check to clear previous residue if any. Now that it is not just the local clear-up, calling logout on something which is known to be logged in is unnecessary.